### DISCUSSION TITLE
SNUtils needs a reload to work

### DISCUSSION THREAD

**Discussion by Sauron-Dev** [arnoudkooi/SN-Utils#321] [Q&A]
I am not sure if I should raise an issue yet, thats why I try it with a Q&A first. On my setup (chrome, macos) when I navigate the first time to our servicenow instance (hosted by servicenow, but using a custom url, so we have to use the onprem plugin) snutils is not working correctly. You can get the snutils box, but when you enter a command or table nothing happens, the box can also not be closed. After you reload the tab snutils works like it should.

Please remove this post when its not the right place to ask such questions :) Thank you very much in advance.

**Comment by arnoudkooi** [comment]
Have you given permission to run on your (sub)domain?

<img width="653" alt="image" src="https://user-images.githubusercontent.com/840781/200566583-8b1f58fd-a72f-44d1-8b66-dfb0138e9df9.png">

You can manage them in the extension settings as well:
<img width="731" alt="image" src="https://user-images.githubusercontent.com/840781/200567086-3a32ee9c-9b6a-4665-9807-968d99121d79.png">

**Reply by Sauron-Dev** [reply]
Yes I have set this settings to "On All Site", so this should not be the issue.

But what I just tested is if this happens in a incognito window aswell (I allowed the snutils extension to run in incognito mode). And there it does not happen.....

so might it be another extension that makes trouble?

**Reply by Sauron-Dev** [reply]
Also my Update Set picker does not go into the extended format (bigger and with the context menu) anymore. Even when snutils is running (after reloading the tab on the first visit) the update set picker does not work (even not with incognito mode)...

**Reply by arnoudkooi** [reply]
You do have the setting enabled? Maybe try a reinstall?
<img width="788" alt="image" src="https://user-images.githubusercontent.com/840781/201683845-a19a412e-b1c1-4432-8b12-e4b953f71e45.png">

**Reply by Sauron-Dev** [reply]
Yeah I do have the settings enabled. My Settings looks a bit different (for example your last point is not present in my settings), I guess thats because we need to use the onPrem version (we use custom URL).

**Reply by arnoudkooi** [reply]
No, this is the upcoming update 6.3 https://youtu.be/oq5EIXqWQ0A

**Reply by Sauron-Dev** [reply]
I just tried a reinstall but it didn't help :/

**Reply by arnoudkooi** [reply]
Have you tried disabling other extensions, and or do you see console errors.

**Reply by Sauron-Dev** [reply]
Yes i tried that and I just did it again. Its odd the thing with the not working at the first visit seems to be not reproducible anymore, but the update set picker still do not work. Also my colleague has the problem that the picker (and more important the context menu) does not work anymore.

**Reply by arnoudkooi** [reply]
We could do a quick call to check. Let me know...

**Reply by arnoudkooi** [reply]
seems an update fixed tis

**Reply by Sauron-Dev** [reply]
Yes we could solve that, thank you very much Arnoud!
